### PR TITLE
Add final price column in products list

### LINKED ISSUE #11479: Show price with TAX on backend product list
Please restore to backend product list prices with TAX column....
This is very annoying complication in B2C businesses, and its cant be fixed by override of src/PrestaShopBundle/ folder

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add a column from PrestaShop 1.6 on product catalog missing on PS 1.7
| Type?         | new feature
| Category?     | BO
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | Fixes #11479
| How to test?  | Column "price tax included" exists and displays data on the products catalog

finalprice

Added product final price with tax column (useful for B2C merchants)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11484)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig [hunk 1]
```diff
@@ -129,7 +132,7 @@
               'value': filter_column_price,
             } %}
           </th>
-
+          <th class="text-center"></th>
```
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig:135]: I guess this line is not needed here, as you do not add values in the array in #11483

**Comment by Quetzacoalt91** [comment]
Linked to #11483

**Comment by Quetzacoalt91** [comment]
Hi @PrestaShark,

I merged your PRs together. Can you please take some time to edit your first message and fill the provided table?
These information are needed to classify your contribution and then help the QA team to validate your changes.

Thanks.

File: src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig (hunk 1 above)
Comment (Quetzacoalt91) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig:135]: Found out this was used for the filter inputs.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by marionf** [comment]
Hello @PrestaShark 

Can you add the min and max input for this column like for price tax excluded ?

capture d ecran_675

**Comment by mickaelandrieu** [comment]
Hello @PrestaShark,

before considering any merge, please update the pull request description as request by @prestonBot :+1: 

Regards,

**Comment by kpodemski** [comment]
Guys, it is very needed in 1.7.5, maybe consider to give this task to someone from the team, i don't think that @PrestaShark will handle filtering by price etc.

**Comment by Quetzacoalt91** [comment]
True. @marionf, the purpose of this PR is to have the same behavior as 1.6. Adding filters can be done in another PR.

However @PrestaShark, please take some time to fill the table in the first message to help us classifying your PR. Thank you.

**Comment by Quetzacoalt91** [comment]
Updated the table by myself. Thanks everybody for review.

**Comment by ghost** [comment]
Sorry about no response, and thanks for Your time. 
I was trying to add working filters localy but without success (inputs are not big problem)

Thanks anyway. Life savers ;)

**Comment by nincris** [comment]
Hi Quetzacoalt91,
Nice work... I use PS 1.7.8.7 and this problem is solved, but it still has a bug... if the product has a discount, then the price shown in the admin is not the final one - the one on the website as it was in PS 1.6. ..
The price should include the discount if the product has one. This way you can see the final price on the website directly in BO

**Pull request merged**